src/pages/Admin/AdminVenuePreview.tsx

```diff
@@ -59,7 +67,10 @@ export const AdminVenuePreview: React.FC<AdminVenuePreview> = ({
           <span className="title">Banner photo</span>
           <img
             className="banner"
-            src={venue.config.landingPageConfig.coverImageUrl}
+            src={
+              venue.config.landingPageConfig.bannerImageUrl ??
```

Comment:
Should we render the image conditionally so that nothing is displayed. Here we'd see the `image missing` artefact with the alt text.

Reply:
basically when the venue is created the banner goes to venue/config/landingPageConfig/checklist/coverImageUrl. However when it is updated it is stored in venue/config/landingPageConfig/bannerImageUrl.

So that line checks if an image exists in bannerImageUrl, if not it means it has not been updated and shows the one in coverImageUrl at the time of creation.

Reply:
That sounds like an error. @maxb94 ?